src/phoenix/experimental/evals/functions/classify.py

```diff
@@ -163,19 +171,21 @@ def _process_response(response: str) -> Tuple[str, Optional[str]]:
                 explanation = None
         return _snap_to_rail(unrailed_label, rails, verbose=verbose), explanation
 
-    async def _run_llm_classification_async(prompt: str) -> Tuple[str, Optional[str]]:
+    async def _run_llm_classification_async(prompt: str) -> Tuple[str, Optional[str], str]:
```

Comment:
suggestion: might be good to type alias this tuple response as the type signature itself doesn't describe the contents.